sunxspex/tests/test_thermal.py

```diff
@@ -400,6 +401,6 @@ def test_energy_out_of_range_warning():
         # Cause all warnings to always be triggered.
         warnings.simplefilter("always")
         # Trigger a warning.
-        output = thermal.line_emission(np.arange(3, 28, 0.5) * u.keV, 6*u.MK, 1e44/u.cm**3)
+        thermal.line_emission(np.arange(3, 28, 0.5) * u.keV, 6*u.MK, 1e44/u.cm**3)
```

Comment:
Deleting this will cause the output of this function to be printed out in some logs which will probably be messy.  Maybe best to keep the variable here even though it's not used.

Reply:
ok I can add a `#noqa` for this line